com.github.petebuffon.yafc.appdata.xml

```diff
@@ -6,12 +6,11 @@
   <summary>YAFC is a Factorio calculator, planner and analyser. The main goal of YAFC is to cope with heavily modded games.</summary>
   <description><p>YAFC is more than just calculator. It contains multiple analysis algorithms and tries to understand what is going on in your modpack. These analyses go as deep as trying to estimate amount of ores generated by the map generator, and as wide as calculating late-game base trying to figure how much of everything you are going to need. It knows which items are important and which are less important, which recipes are efficient and which are less efficient. YAFC handles recursive recipes extremely well, it uses Google OrTools as its model solver.</p><p>YAFC also includes several tools, the most important currently is "Never Enough Items" - it is FNEI on steroids, showing not only recipes to produce or consume an item, but also which recipes you probably want to use, and how much.</p><p>NOTE: This package is not verified by, affiliated with, or supported by ShadowTheAge, the original creator of YAFC..</p></description>
   <url type="homepage">https://github.com/ShadowTheAge/yafc</url>
-  <screenshots>
-    <screenshot type="default">https://raw.githubusercontent.com/flathub/com.github.petebuffon.yafc/master/screenshots/demo_page.png</screenshot>
-  </screenshots>
+  <developer_name>ShadowTheAge</developer_name>
```

Comment:
I think you need to make up your mind about this. Either you're the developer or the original author is and app ID needs to be changed.

Reply:
developer_name was changed to petebuffon